src/main/webapp/app/exam/manage/students/verify-attendance-check/exam-students-attendance-check.component.html

```diff
@@ -0,0 +1,85 @@
+    <br />
+    <div class="table-responsive" id="exams-table" *ngIf="exam">
+        <table class="table table-striped">
+            <thead>
+                <tr jhiSort [(predicate)]="predicate" [(ascending)]="ascending" (sortChange)="sortRows()">
+                    <th class="d-md-table-cell" jhiSortBy="id">
+                        <span>{{ 'global.field.id' | artemisTranslate }}</span>
+                        <fa-icon [icon]="faSort"></fa-icon>
+                    </th>
+                    <th></th>
+                    <th class="d-md-table-cell" jhiSortBy="login">
+                        <span>{{ 'artemisApp.examManagement.examStudents.login' | artemisTranslate }}</span>
+                        <fa-icon [icon]="faSort"></fa-icon>
+                    </th>
+                    <th class="d-md-table-cell" jhiSortBy="registrationNumber">
+                        <span>{{ 'artemisApp.examManagement.examStudents.registrationNumber' | artemisTranslate }}</span>
+                        <fa-icon [icon]="faSort"></fa-icon>
+                    </th>
+                    <th class="d-md-table-cell">
+                        <span>{{ 'artemisApp.examManagement.examStudents.signingImagePath' | artemisTranslate }}</span>
+                    </th>
+                    <th class="d-md-table-cell">
+                        <span>{{ 'artemisApp.studentExams.started' | artemisTranslate }}</span>
+                    </th>
+                    <th class="d-md-table-cell">
+                        <span>{{ 'artemisApp.studentExams.submitted' | artemisTranslate }}</span>
+                    </th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr *ngFor="let examUser of allExamUsersAttendanceCheck">
+                    <td class="d-md-table-cell">
+                        <span>
+                            {{ examUser.id }}
+                        </span>
+                    </td>
+                    <td class="d-md-table-cell">
+                        <img class="img-fluid" [width]="60" [src]="examUser.studentImagePath ? SERVER_API_URL + examUser.studentImagePath : missingImage" />
+                    </td>
+                    <td class="d-md-table-cell">
+                        <span>
+                            {{ examUser.login }}
+                        </span>
+                    </td>
+                    <td class="d-md-table-cell">
+                        <span>
+                            {{ examUser.registrationNumber }}
+                        </span>
+                    </td>
+                    <td class="d-md-table-cell">
+                        <img *ngIf="examUser?.signingImagePath" class="img-fluid" [width]="60" [src]="SERVER_API_URL + examUser.signingImagePath" />
+                        <h3 *ngIf="!examUser?.signingImagePath">
+                            <span class="m-2">
+                                <fa-icon [ngStyle]="{ color: 'red' }" [icon]="faXmark"></fa-icon>
```

Comment:
This doesn't need to be a computed property if it is static. Use a Boostrap color class instead, please.